### DISCUSSION TITLE
How to add my own os image

### DISCUSSION THREAD

**Discussion by manojdeshmukh45** [copy/v86#781] [Q&A]
When I tried to install in my own server, how can I use my own custom os image.
where i need to add some of the own files in it.

**Comment by Darin755** [comment]
I created a personal project called browser Linux that aims to take away the complexity of v86.

https://github.com/Darin755/browser-linux


All you need is a simple web server. You can swap out the default image with your own by simply naming your iso "rootfs.iso"
